test_framework/terraform/aws/sle-micro/user-data-scripts/provision_k3s_agent.sh.tpl

```diff
@@ -18,5 +18,5 @@ elif [ -b "/dev/xvdh" ]; then
   sudo mount /dev/xvdh /var/lib/longhorn
 fi
 
-curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="agent --token ${k3s_cluster_secret}" K3S_URL="${k3s_server_url}" INSTALL_K3S_VERSION="${k3s_version}" sh -
+sudo sh -c 'curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="agent --token ${k3s_cluster_secret}" K3S_URL="${k3s_server_url}" INSTALL_K3S_VERSION="${k3s_version}" sh -'
```

Comment:
It looks verbose.

Can you help to try this instead:
curl -sfL https://get.k3s.io | **sudo** NSTALL_K3S_EXEC="agent --token ${k3s_cluster_secret}" K3S_URL="${k3s_server_url}" INSTALL_K3S_VERSION="${k3s_version}" sh -

Reply:
Hi @yangchiu :
    I have tested it by running the Terraform scripts and updated my PR.
![Screenshot_20230611_163342](https://github.com/longhorn/longhorn-tests/assets/104337648/c92f3b8e-93b9-4c60-ad44-eba99867d5bc)